SofaKernel/modules/SofaBaseVisual/VisualModelImpl.cpp

```diff
@@ -1515,16 +1514,16 @@ void VisualModelImpl::handleTopologyChange()
 
                                 if(!is_in_shell)
                                 {
-                                    sout << "INFO_print : Vis - triangle is forgotten in SHELL !!! global indices (point, triangle) = ( "  << last << " , " << ind_forgotten  << " )" << sendl;
+                                    msg_info() << "INFO_print : Vis - triangle is forgotten in SHELL !!! global indices (point, triangle) = ( "  << last << " , " << ind_forgotten  << " )";
 
                                     if(ind_forgotten<m_topology->getNbTriangles())
                                     {
                                         const core::topology::BaseMeshTopology::Triangle t_forgotten = m_topology->getTriangle(ind_forgotten);
-                                        sout << "INFO_print : Vis - last = " << last << sendl;
-                                        sout << "INFO_print : Vis - lastIndexVec[i] = " << lastIndexVec[i] << sendl;
-                                        sout << "INFO_print : Vis - tab.size() = " << tab.size() << " , tab = " << tab << sendl;
-                                        sout << "INFO_print : Vis - t_local rectified = " << triangles[j_loc] << sendl;
-                                        sout << "INFO_print : Vis - t_global = " << t_forgotten << sendl;
+                                        msg_info() << "INFO_print : Vis - last = " << last;
+                                        msg_info() << "INFO_print : Vis - lastIndexVec[i] = " << lastIndexVec[i];
+                                        msg_info() << "INFO_print : Vis - tab.size() = " << tab.size() << " , tab = " << tab;
+                                        msg_info() << "INFO_print : Vis - t_local rectified = " << triangles[j_loc];
+                                        msg_info() << "INFO_print : Vis - t_global = " << t_forgotten;
```

Comment:
The proper way for this kind of message is:
```cpp
msg_info() << "Vis - last = " << last << msgendl
                  << "Vis - lastIndexVec[i] = " << lastIndexVec[i] << msgendl
                  << Vis - tab.size() = " << tab.size() << " , tab = "  << msgendl 
                  ....
```  

Doing this, emit only one message which allow correct message visualization and reporting in the GUI (eg in the message windows of GUI runSofa2). It is also more efficient as the cost of message processing is divided by 5.